Bool tf_node_rdy